action.yml

```diff
@@ -14,9 +14,9 @@ inputs:
     required: false
     default: 'jdk'
   architecture:
-    description: 'The architecture of the package'
+    description: 'The architecture of the package (defaults to the action runner's architecture)'
```

Comment:
I think you should use double quotes.

Reply:
No. I think you need to use double double quotes because you use single quote in `runner's architecture` that is why I think it has issues.

Reply:
> No. I think you need to use double double quotes because you use single quote in `runner's architecture` that is why I think it has issues.

Oh, yes. Haha, thanks!